updating/updating-restricted-network-cluster/index.adoc

```diff
@@ -13,6 +13,11 @@ If the local container registry and the cluster are connected to the mirror regi
 
 A single container image registry is sufficient to host mirrored images for several clusters in the disconnected network.
 
+== Mirroring the OpenShift Container Platform image repository
+In order to perform an update in a disconnected environment, your cluster environment must have access to a mirror registry that contains the necessary images and resources for your targeted update. The following page contains instructions for mirroring images onto a repository in your disconnected cluster:
+
+* xref:../../updating/updating-restricted-network-cluster/mirroring-image-repository.adoc#mirroring-ocp-image-repository[Mirroring the OpenShift Container Platform image repository]
```

Comment:
Same here about using {product-title}

Reply:
This is referring to the hyperlink text in line 19, right? If so, will do!

Just to make sure: we can use attributes in headings and hyperlink text, right? Is there anywhere besides the topic map where we can't or shouldn't use attributes?

Reply:
Yep you can use them in xrefs/links, etc. Pretty much anywhere in an .adoc file you should be able to use them. It's only in a non-.adoc file (like _topic_map.yml) that you can't.